### PR TITLE
Added date folder to S3

### PR DESCRIPTION
**Pull request by noppanit**
#23  

This one should be merged with #26 because of the way we returned the URL is changed.

### CODE REVIEW

**Comment by noppanit** [comment]
@mpatek #reviewmerge please.

File: src/config_parser.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 import os
 import functools
 import json
+import datetime
```
Comment (mpatek) [inline src/config_parser.py]: @noppanit why are you importing `datetime` here. Are you using it in this module?

File: src/date_manager.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import datetime
+
+def get_current_date():
+    return datetime.date.today().strftime('%Y-%m-%d')
```
Comment (mpatek) [inline src/date_manager.py]: Instead of using `strftime`, you might want to just use `isoformat()`

File: src/config_parser.py (hunk 1 above)
Comment (noppanit) [inline src/config_parser.py]: sorry bad cleanup

**Comment by mpatek** [comment]
@noppanit looks good. Just a couple of small things.

**Comment by noppanit** [comment]
@mpatek Fixed thanks for the `isoformat` I didn't know about that before.

File: src/config_parser.py (hunk 1 above)
Comment (mpatek) [inline src/config_parser.py]: You might consider adding linting to this repo. We're using something called `flake8` in Glance, that works pretty well. You can even add it to the Travis CI tests: https://github.com/fusioneng/glance-app/blob/master/.travis.yml#L8

**Comment by mpatek** [comment]
Thanks @noppanit

**Pull request merged**